app/javascript/plugins/init_mapbox.js

```diff
@@ -9,9 +9,11 @@ const buildMap = (mapElement) => {
 };
 
 const fitMapToMarkers = (map, markers) => {
-  const bounds = new mapboxgl.LngLatBounds();
-  markers.forEach(marker => bounds.extend([marker.lng, marker.lat]));
-  map.fitBounds(bounds, { padding: 70, maxZoom: 15, duration: 0 });
+  if (markers.length > 0) {
```

Comment:
Is this so it fits in the map?

Reply:
It is something Gianluca added when map wasnt displaying when there is no lng or lat which resulted in error